frontend/src/pages/GamePage.tsx

```diff
@@ -1,27 +1,83 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Terminal, { ColorMode, TerminalOutput } from "react-terminal-ui";
-import { quizQuestions } from "./questions";
+
+interface Option {
+    text: string;
+    points: {
+        health: number;
+        wealth: number;
+        happiness: number;
+    };
+}
+
+interface Question {
+    title: string;
+    options: Option[];
+}
 
 const GamePage: React.FC = () => {
-    const [terminalLineData, setTerminalLineData] = useState([
-        <TerminalOutput>Welcome to the Life Choices Game!</TerminalOutput>,
-        <br />,
-        <TerminalOutput>Your initial stats: Health: 50, Happiness: 50, Money: 50.</TerminalOutput>,
-        <br />,
-        <TerminalOutput>{`Question 1: ${quizQuestions[0].question}`}</TerminalOutput>,
-        ...quizQuestions[0].options.map((option, index) => (
-            <TerminalOutput key={`tout-${index}`}>{`${index + 1}. ${option.text}`}</TerminalOutput>
-        )),
+    const [questions, setQuestions] = useState<Question[]>([]);
+    const [terminalLineData, setTerminalLineData] = useState<string[]>([
+        "Welcome to the Life Choices Game!",
+        "\nYour initial stats: Health: 50, Wealth: 50, Happiness: 50.\n",
     ]);
-    const [currentQuestionIndex, setCurrentQuestionIndex] = useState<number>(0);
-    const [stats, setStats] = useState<{ health: number; happiness: number; money: number }>({
+    const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
+    const [stats, setStats] = useState<{ health: number; wealth: number; happiness: number }>({
         health: 50,
+        wealth: 50,
         happiness: 50,
-        money: 50,
     });
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState<string | null>(null);
+    const [gameOver, setGameOver] = useState(false);
+
+    // Fetch questions from the backend
+    useEffect(() => {
+        const fetchQuestions = async () => {
+            try {
+                const response = await fetch("https://kernel-panic.onrender.com/api/getAllQuestions");
```

Comment:
Use the PanicApi instance instead of using fetch for calling api.

Reply:
Instead of this line
```
const response = await fetch("https://kernel-panic.onrender.com/api/getAllQuestions");
```
write:
```
const response = await PanicApi.get("/getAllQuestions");
```